server/update_server.sh

```diff
@@ -16,4 +16,21 @@ for f in server/redis-migrations/*.js; do
   node -r "./babel-register" -r "core-js/proposals/reflect-metadata" -r "dotenv/config" "$f" || exit 1
 done
 
+echo "Syncing public assets to the cloud"
```

Comment:
This should probably only run if Spaces is configured, otherwise this can fail in weird ways